pkgs/by-name/pc/pcsx2/linux.nix

```diff
@@ -55,23 +54,12 @@ llvmPackages_17.stdenv.mkDerivation (finalAttrs: {
   src = fetchFromGitHub {
     owner = "PCSX2";
     repo = "pcsx2";
-    fetchSubmodules = true;
     rev = "v${finalAttrs.version}";
```

Comment:
```suggestion
    rev = "refs/tags/v${finalAttrs.version}";
```

Reply:
What is the meaningful difference of adding `refs/tags/` here? (Since `v${version}` works.)

Reply:
That refs/tags/ is not ambiguous, just using v${version} could also mean a branch.